pom.xml

```diff
@@ -58,6 +58,11 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
+      <version>6.2.10</version>
```

Comment:
Assumed I didn't need to define new version properties for the 3rd party transitive libs having their versions 'fixed' as this wasn't done for some existing libs (e.g. `tomcat-embed-core`).

Reply:
The reason: it highlights which dependencies are temporary and should be removable at a later date.